### PR TITLE
🚦 Add FontAwesome to Project

### LINKED ISSUE #29: Add fontawesome for icons
Use https://fontawesome.com/ package and for icons

### PR DESCRIPTION
**Pull request by shrihari689**
## Proposed Changes ⏰

- Add FontAwesome Webfonts
- Move Stylesheets from JS Pack folder to assets folder.
- Tested Some Icons

## Merge Checklist 📜

- [✅] Add specs that demonstrate bug / test a new feature.
- [✅ ] Check if route, query, or mutation authorization looks correct.
  - Add tests for authorization, if required.
- [💡] Update docs, where applicable.
- [💡 ] Add development seeds for new tables.

📌 Closes #29

### CODE REVIEW

**Comment by shrihari689** [comment]
## Additional Changes ⏰

- Add FontAwesome JS 🌟
- Replace all SVGs with FA Icons. 🔨
- Tested ✅

@bodhish  @arnabsen1729 👀

**Comment by bodhish** [comment]
We don't need that animated icons and fonts.

We use the FA js pack to replace the `<i>` elements with `svg`

Using svg gives a superior quality render compared to fonts.


Also remove all the custom styles added. 😅

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Tailwind
 @import "tailwindcss/base";
 @import "tailwindcss/components";
-@import "tailwindcss/utilities";
+@import "tailwindcss/utilities";
```
Comment (bodhish) [inline app/assets/stylesheets/application.scss:4]: The location should be ` app/javascript/stylesheets/application.scss`

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -9,10 +9,11 @@ import * as ActiveStorage from "@rails/activestorage";
 import "channels";
 import * as PatientFormPage from "../Patient/New/PersonalDetails/PersonalDetails.bs";
 
+import "./../../assets/stylesheets/application";
```
Comment (bodhish) [inline app/javascript/packs/application.js:12]:
```suggestion
import "stylesheets/application";
```

**Review by bodhish — APPROVED** [review]

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (shrihari689) [reply app/assets/stylesheets/application.scss]: Restored our `application.scss` back to `JS/Stylesheets`

**Pull request merged**